stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/ChatTheme.kt

```diff
@@ -29,6 +30,10 @@ private val LocalReactionTypes = compositionLocalOf<Map<String, Int>> {
     error("No reactions provided! Make sure to wrap all usages of Stream components in a ChatTheme.")
 }
 
+private val LocalStreamConfiguration = compositionLocalOf<StreamConfiguration> {
+    error("No stream configuration provided! Make sure to wrap all usages of Stream components in a ChatTheme.")
```

Comment:
Nitpicking , shouldn't `stream` be `Stream` ? branding and all

Reply:
As it's a class name, I'd go with
```suggestion
    error("No StreamConfiguration provided! Make sure to wrap all usages of Stream components in a ChatTheme.")
```

Reply:
Will update!